completion/graphicsonthefly.cwl

```diff
@@ -0,0 +1,107 @@
+# graphicsonthefly package
+# ponte-vecchio 2023-01-17 for v0.9.6
+
+#include: graphicx
+
+#keyvals:\usepackage/graphicsonthefly#c
+renewall
+animate
+all
+curl
+nocoalesce
+convert
+#endkeyvals
+
+## Animations
+#ifOption:all
+#include: animate
+\animatedgifonthefly[%<options%>%keyvals]{%<fps%>}{%<image name%>}{%<start%>}{%<end%>}{%<url%>}
+\animatedgif{%<fps%>}{%<image name%>}{%<start%>}{%<end%>}{%<url%>}
+\usegifonthefly[%<options%>%keyvals]{%<image name%>}{%<url%>}
+#endif
+
+#ifOption:animate
+#include: animate
+\animatedgifonthefly[%<options%>%keyvals]{%<fps%>}{%<image name%>}{%<start%>}{%<end%>}{%<url%>}
+\animatedgif{%<fps%>}{%<image name%>}{%<start%>}{%<end%>}{%<url%>}
+\usegifonthefly[%<options%>%keyvals]{%<image name%>}{%<url%>}
+#endif
+
+#keyvals:\animatedgifonthefly#c, \usegifonthefly#c
```

Comment:
Similarly the space needs to be removed here, it should be `#keyvals:\animatedgifonthefly#c,\usegifonthefly#c`

Reply:
Reflected in the new commit with all other cwl files, grep saves the day :)